src/open_inwoner/scss/components/UserFeed/UserFeed.scss

```diff
@@ -1,5 +1,6 @@
 .userfeed {
   position: relative;
+  margin-top: var(--row-height);
```

Comment:
Yes, this is alright for now since the other plugins are also using a margin and these are only used on Home - but actually... 
I would need to refactor/check them all at some point, since "Margins break component encapsulation. A well-built component should not affect anything outside itself... Components shouldn't care about spacing between them. Their parents should."